tests/tests_fabric/utilities/test_device_parser.py

```diff
@@ -38,7 +38,6 @@ def test_determine_root_gpu_device(devices, expected_root_gpu):
 @pytest.mark.parametrize(
     ["devices", "expected_gpu_ids"],
     [
-        (None, None),
```

Comment:
We don't need to support None anymore. This was for the time when `devices=None` was valid (before auto)